### ISSUE TITLE
Setting the value of date to "today" breaks the form (investigating)

### ISSUE THREAD

**Issue by joeribekker** [open-formulieren/open-forms#2828]

**Comment by joeribekker** [comment]
@LaurensBurger elaborate pleaser

**Comment by LaurensBurger** [comment]
when using the following logic to set the date in a date component
```json
{
  "component": "",
  "variable": "datecomponent",
  "formStep": null,
  "formStepUuid": null,
  "action": {
    "type": "variable",
    "value": {
      "var": "today"
    }
  }
}
```
the form crashes
```
TypeError /api/v2/submissions/{submission_uuid}/steps/{step_uuid}
fromisoformat: argument must be str
```

**Comment by sergei-maertens** [comment]
Fixed by accident via a9f2c90abe8cd9ae95af2386efbaa5909a39423e
